ProjectLighthouse/Types/Serialization/GameManualXml.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Xml.Serialization;
+
+namespace LBPUnion.ProjectLighthouse.Types.Serialization;
+
+public class GameManualXml : ILbpSerializable
+{
+    [XmlText]
```

Comment:
```suggestion
```
This doesn't actually need the XmlText since it doesn't get reflected by the serializer at all